environments/ops-engineering.json

```diff
@@ -1,14 +1,6 @@
 {
```

Comment:
This whole file can be deleted as the development environment is the last one

Reply:
File deleted